src/design/plone/policy/tests/test_initial_structure.py

```diff
@@ -0,0 +1,176 @@
+
+    def test_amministrazione_section(self):
+
+        amministrazione = self.portal['amministrazione']
+        self.assertEqual(amministrazione.constrain_types_mode, 1)
+        self.assertEqual(amministrazione.locally_allowed_types, ("Document",))
+
+        self.assertEqual(
+            amministrazione.keys(),
+            [
+                'politici',
+                'personale-amministrativo',
+                'organi-di-governo',
+                'aree-amministrative',
+                'uffici',
+                'enti-e-fondazioni',
+                'luoghi',
+            ],
+        )
+
+        self.assertEqual(amministrazione['politici'].portal_type, 'Document')
+        self.assertEqual(amministrazione['politici'].constrain_types_mode, 1)
+        self.assertEqual(
+            amministrazione['politici'].locally_allowed_types, ("Document",)
+        )
+
+        self.assertEqual(
+            amministrazione['personale-amministrativo'].portal_type, 'Document'
+        )
+        self.assertEqual(
+            amministrazione['personale-amministrativo'].constrain_types_mode, 1
+        )
+        self.assertEqual(
+            amministrazione['personale-amministrativo'].locally_allowed_types,
+            ("Persona",),
+        )
+
+        self.assertEqual(
+            amministrazione['organi-di-governo'].portal_type, 'Document'
+        )
+        self.assertEqual(
+            amministrazione['organi-di-governo'].constrain_types_mode, 1
+        )
+        self.assertEqual(
+            amministrazione['organi-di-governo'].locally_allowed_types,
+            ("UnitaOrganizzativa",),
+        )
+
+        self.assertEqual(
+            amministrazione['aree-amministrative'].portal_type, 'Document'
+        )
+        self.assertEqual(
+            amministrazione['aree-amministrative'].constrain_types_mode, 1
+        )
+        self.assertEqual(
+            amministrazione['aree-amministrative'].locally_allowed_types,
+            ("UnitaOrganizzativa",),
+        )
+
+        self.assertEqual(amministrazione['uffici'].portal_type, 'Document')
+        self.assertEqual(amministrazione['uffici'].constrain_types_mode, 1)
+        self.assertEqual(
+            amministrazione['uffici'].locally_allowed_types,
+            ("UnitaOrganizzativa",),
+        )
+
+        self.assertEqual(
+            amministrazione['enti-e-fondazioni'].portal_type, 'Document'
+        )
+        self.assertEqual(
+            amministrazione['enti-e-fondazioni'].constrain_types_mode, 1
+        )
+        self.assertEqual(
+            amministrazione['enti-e-fondazioni'].locally_allowed_types,
+            ("UnitaOrganizzativa",),
+        )
+
+        self.assertEqual(amministrazione['luoghi'].portal_type, 'Document')
+        self.assertEqual(amministrazione['luoghi'].constrain_types_mode, 1)
+        self.assertEqual(
+            amministrazione['luoghi'].locally_allowed_types, ("Venue",)
+        )
+
+    def test_servizi_section(self):
+
```

Comment:
Bisogna chiarirsi con Baio e Ste: dicevano che può essere fuorviante mettere i servizi dentro dei folder sotto /servizi finché non si capisce bene dai listing che tipi si stanno guardando. In effetti se non sai cosa stai guardando non è facile distinguere che sotto /servizi hai delle cartelle e non dei servizi

Reply:
Questo ovviamente cambierebbe anche il test. Una fix può essere permettere di usare Servizi e Document sotto /servizi